### PR TITLE
Upadhyaydhruv/parser

### PR DESCRIPTION
**Pull request by upadhyaydhruv**
Implements parser, API for flask

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,12 +7,17 @@
     "@testing-library/react": "^11.2.3",
     "@testing-library/user-event": "^12.6.0",
     "bootstrap": "^4.5.3",
+<<<<<<< HEAD
+    "node-fetch": "^2.6.1",
+=======
     "firebase": "^8.2.3",
+>>>>>>> origin
```
Comment (sahil-kale) [inline package.json]: Committed Merge conflicts lmao

File: parser/affinda.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+global.fetch = require("node-fetch");
+
+let token = '9f9cc8a21bb32ad7e5e2036f919346c9863b71e1'
```
Comment (sahil-kale) [inline parser/affinda.js]: committed token

File: parser/test_affinda.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+import flask
+from flask.json import jsonify
+
+
+def return_json(path):
+    FILE_TO_UPLOAD_PATH = Path(path)
+
+    url = "https://resume-parser.affinda.com/public/api/v1/documents/"
+
+    token = "9f9cc8a21bb32ad7e5e2036f919346c9863b71e1"
```
Comment (sahil-kale) [inline parser/test_affinda.py]: committed token

File: parser/test_js.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+global.fetch = require("node-fetch");
+const ResumeParser = require('resume-parser');
+
+ResumeParser
+  .parseResumeFile('/home/goodmanmcgee/JobSearchAutomator/parser/Resume_2021.pdf') // input file, output dir
```
Comment (sahil-kale) [inline parser/test_js.js]: can you make these paths generic strings that only need to be updated once?

File: parser/test_sovren.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+from pdfminer.high_level import extract_text
+sys.path.append("/usr/home/username/pdfminer")
+
+
+def extract_text_from_pdf(pdf_path):
+    return extract_text(pdf_path)
+
+
+if __name__ == '__main__':
+    print(extract_text_from_pdf('/home/goodmanmcgee/JobSearchAutomator/parser/Resume_2021.pdf'))  # noqa: T001
```
Comment (sahil-kale) [inline parser/test_sovren.py]: See above

**Review by sahil-kale — CHANGES_REQUESTED** [review]
Can the resumes be gitignored so that they are not stored on remote?

File: parser/token.key [hunk 6]
```diff
@@ -0,0 +1 @@
+f92f792039d60b67790b37e6db90ce3a26d4803b
```
Comment (sahil-kale) [inline parser/token.key]: committed key lol

**Review by sahil-kale — CHANGES_REQUESTED** [review]

**Comment by upadhyaydhruv** [comment]
Hee Hee Hee!!

**Pull request merged**